insights/tests/client/test_archive.py

```diff
@@ -34,11 +36,21 @@ def test_init_archive(self, cleanup, register, mkdtemp):
         assert archive.archive_name == test_archive_name
 
         cleanup.assert_called_once()
-        mkdtemp.assert_has_calls([call(dir='/var/tmp/', prefix='insights-archive-')])
+        mkdtemp.assert_has_calls([call(dir=constants.insights_tmp_path, prefix='insights-archive-')])
         register.assert_called_once()
 
+    @patch('insights.client.archive.os.mkdir')
+    @patch('insights.client.archive.os.path.exists', Mock(side_effect=[False]))
+    def test_create_tmp_dir(self, mkdir, _, __):
+        '''
+        Verify temporary director insights_tmp_path is created if not exists
```

Comment:
Style improvement suggestion.

```suggestion
        Verify temporary director insights_tmp_path is created if it does not exist
```